tough/src/error.rs

```diff
@@ -420,7 +429,16 @@ pub enum Error {
     TargetNotFound { target_url: String },
 
     #[snafu(display("Delegated role not found: {}", name))]
-    DelegateNotFound {
+    DelegateNotFound { name: String },
+
+    #[snafu(display("Targets role not found: {}", name))]
```

Comment:
Errors that have an underlying `source` should display it (unless it's just going to say the exact same thing I guess).

```suggestion
    #[snafu(display("Targets role '{}' not found: {}", name, source))]
```